src/main/java/com/nashtech/rookie/yasa/controller/GlobalExceptionHandler.java

```diff
@@ -17,12 +19,41 @@ public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
                 ProductNotFoundException.class,
                 RatingNotFoundException.class,
                 UserNotFoundException.class,
-                CartNotFoundException.class
+                CartNotFoundException.class,
```

Comment:
Exceptions should not be divided according to each model. We should write code generically. We should just divide exceptions by error type